### ISSUE TITLE
AXE Core test - expose "needs review" results

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2020]
Scope
---
Automated Accessibility Testing with AXE Core is implemented in PDS Storefront. Results are only exposed in a file if there are any real issues detected (called: violations). But very often, results include also a status of "needs review" (called: inapplicable) which the rules engine can't resolve by itself (if it is a real issue or not). Currently those results are ignored completely. But we must also take care about those results and find ways how to store/expose them.

Out of Scope
---
- Implementation of flagging feature of "needs review" results to prevent reoccur of those results if they were proven to be not an issue

Notes  
---
- In AXE Core "needs review" results are flagged with "inapplicable"
- Check different AXE Reporter Variants in configure option: https://www.deque.com/axe/core-documentation/api-documentation/ 

Acceptance criteria
---
- [ ] "Needs Review" results must be exposed in a way to help developers to find and review them

Subtasks
---
- [ ] Task
